Doc/using/mac.rst

```diff
@@ -125,13 +125,11 @@ http://www.hashcollision.org/hkn/python/idle_intro/index.html.
 Installing Additional Python Packages
 =====================================
 
-There are several methods to install additional Python packages:
+.. note::
```

Comment:
I don't think that it should be displayed as a note.

Reply:
Done (on mobile, terrible interface for applying changes!)